prompb/buf.yaml

```diff
@@ -0,0 +1,4 @@
+version: v1
+name: buf.build/prometheus/prometheus
```

Comment:
@roidelapluie this repository doesn't exist yet on the BSR, could you create it?